src/common/evaluation/QA/eval_depthmap_models/src/evaluate.py

```diff
@@ -174,6 +177,17 @@ def get_prediction(model_path: str, dataset_evaluation: tf.data.Dataset) -> np.a
     return prediction_list
 
 
+def get_predictions(model_paths: list, dataset_evaluation: tf.data.Dataset) -> list:
```

Comment:
The function name can be more distinctive to `get_prediction()`

```suggestion
def get_predictions_from_multiple_models(model_paths: list, dataset_evaluation: tf.data.Dataset) -> list:
```